packages/sequelize-model/test/migrations/20191105141903_createMinisterVoteTable.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
```

Comment:
if I understand right, this file is now redundant, as we have the migrations defined on root package level